EDDiscovery/Forms/DownloadManagerForm.cs

```diff
@@ -138,6 +138,8 @@ private void DownloadManagerForm_FormClosing(object sender, FormClosingEventArgs
         {
             if (CheckThread != null && CheckThread.IsAlive)     // can't close if its alive, it will call back nothing
                 CheckThread.Join();
+
+            font?.Dispose();
```

Comment:
831 got merged in despite commentary otherwise. Need to reconcile the two, and only do this once.